boards/arm/cy8cproto_063_ble/board.cmake

```diff
@@ -5,3 +5,6 @@
 #***************************************************************************
 
 include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+
+board_runner_args(pyocd "--target=cy8c6xx7_nosmif")
+include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
```

Comment:
These changes are unrelated to overlays and need to go in a separate commit.

Reply:
Split into 2 commits